app/controllers/admin_controller.rb

```diff
@@ -28,4 +29,12 @@ def technologies
   def claims
     @claims = ClaimRequest.all
   end
+
+  private
+
+  def authenticate_admin!
+    if !(user_signed_in? && current_user.admin?)
```

Comment:
Small improvement idea. Refactor `current_user.admin?` to a method in `ApplicationController`:
```ruby
def user_is_admin?
  current_user.admin?
end
```